fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -113,6 +123,25 @@
 
     private static final String ROW_ID = "ROW_ID";
 
+    // A set of resource type names for resources which can be addressed using a
+    // canonical reference (and therefore should have a url attribute and optional version
+    // TODO do we need to do this?
```

Comment:
how to resolve the TODO?

Reply:
Not used, so removed this member and comment.